src/pseudopeople/interface.py

```diff
@@ -116,6 +116,7 @@ def _load_data_from_path(data_path: Path, year_filter: Dict[str, List]):
 
 def _reformat_dates_for_noising(data: pd.DataFrame, dataset: Dataset):
     """Formats SSA event_date and dates of birth, so they can be noised."""
+    data = data.copy()
```

Comment:
Was this the source of the noising not happening? Based on your comment in the summary it seems like this might not be necessary?

Reply:
It wasn't this - it was just a small number thing (hence my increasing all probabilities to 25%).

This _is_ what was causing data to not equal that copy as we saw on zoom, though